defaultsfile/defaults_file.go

```diff
@@ -0,0 +1,110 @@
+// pmm-agent
+// Copyright 2019 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package defaultsfile provides managing of defaults file.
+package defaultsfile
+
+import (
+	"fmt"
+	"os/user"
+	"path/filepath"
+	"strings"
+
+	"github.com/percona/pmm/api/agentpb"
+	"github.com/percona/pmm/api/inventorypb"
+	"github.com/pkg/errors"
+	"gopkg.in/ini.v1"
+)
+
+// DefaultsFile is a struct with database specs fetched from file.
+type DefaultsFile struct {
+	username string
+	password string
+	host     string
+	port     uint32
+}
+
+// New creates new DefaultsFile.
+func New() *DefaultsFile {
+	return &DefaultsFile{}
+}
+
+// ParseDefaultsFile parses given defaultsFile. It returns the database specs.
+func (d *DefaultsFile) ParseDefaultsFile(req *agentpb.ParseDefaultsFileRequest) *agentpb.ParseDefaultsFileResponse {
```

Comment:
Done. 
Just created "Parser" struct because package is "defaultsfile" and with DefaultsFileParser linter raises error about stuttering. ```type name will be used as defaultsfile.DefaultsFileParser by other packages, and that stutters```